scripts/meta

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+# vim: noexpandtab:tabstop=4:shiftwidth=4:fileencoding=UTF-8
+#
+# @author Sebastian Bachmann <sebastian.bachmann@tuwien.ac.at>
+# @desc This script is used to organize scripts
+# @date 10.05.2016
+# @tags
+# @system all
+#
+# Comments in the Beginning of a script define some things and are parsed by this meta script
+# The aim of this script is to find the right script in no time!
+#
+# The Metatag System of Epydoc is used (http://epydoc.sourceforge.net/manual-fields.html#module-metadata-variables)
+# and extendet with the following special tags:
+# 
+# MetaTag					Meaning
+# ----------------------------------------------------------
+# @date						The Date when the script was created, use DD.MM.YYYY as format
```

Comment:
avoid usage of tabs (wrongly displayed on github).

Can you create a README.md for how to tag scripts?